frontends/main/src/app/styled.tsx

```diff
@@ -9,7 +9,7 @@ import { styled } from "ol-components"
  */
 
 export const PageWrapper = styled.div({
-  height: "calc(100vh - 80px)",
+  height: "100vh",
```

Comment:
@gumaerc you're well versed on this it seems! Let's make sure this change doesn't interfere with your more considered [PR](https://github.com/mitodl/mit-open/pull/1685/files).